packages/haul-cli/src/commands/start.ts

```diff
@@ -22,7 +21,6 @@ export default function startCommand(runtime: Runtime) {
     builder: {
       port: {
         description: 'Port to run your webpack server',
-        default: DEFAULT_PORT,
```

Comment:
why removing from here? I guess it's still normalized later, but this feels like a proper place for a default port

Reply:
The thing is if I add it here the `port` from `haul.config.js` won't be picked up. Also we use `port` from the normalised config and there's a fallback to `DEFAULT_PORT` there as well:
```
port: env.port || get(projectConfig, 'server.port', DEFAULT_PORT),
```

Reply:
Ah yea that's a fair call